packages/engine/lib/orchestrator/src/process/local.rs

```diff
@@ -30,6 +30,9 @@ pub struct LocalProcess {
 #[async_trait]
 impl process::Process for LocalProcess {
     async fn exit_and_cleanup(mut self: Box<Self>) -> Result<()> {
+        // Let the engine cleanup some resources
+        tokio::time::sleep(tokio::time::Duration::from_millis(500)).await;
+
```

Comment:
I'm not happy about this, this will add 0.5 seconds to *every* integration test case resulting in a much longer test phase.
Instead, we should properly wait on the Python runner to finish.

Reply:
What should happen when it doesn't respond for more than a minute?

Reply:
We still can kill it, but we want to clean up the experiment anyway